<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QMainWindow" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clicker</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Ресурсы/small_black.png</normaloff>../../Ресурсы/small_black.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
	border: 1px solid rgb(0, 170, 0);
	border-radius: 5px;
}

QLabel {
	font: 12pt &quot;Arial&quot;;
	color: rgb(0, 0, 127);
}

QPushButton {
	padding: .7em 1.2em;
	color: rgb(85, 0, 127);
	font: 75 bold 12pt &quot;Tahoma&quot;;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox_cursor">
      <property name="title">
       <string>Cursor</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPushButton" name="GET_POS">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>POSITION</string>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_pos">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_x">
           <item>
            <widget class="QLabel" name="label_x">
             <property name="text">
              <string>X</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="pos_X">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>12</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>800</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_y">
           <item>
            <widget class="QLabel" name="label_y">
             <property name="text">
              <string>Y</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="pos_Y">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>12</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>500</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_key">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Key name</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="key_name">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="sizeIncrement">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>800</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_click">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Test clicker</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="GET_CLICK">
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string> Click</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>small_green.png</normaloff>small_green.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="click_sum">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>GET_POS</sender>
   <signal>clicked()</signal>
   <receiver>pos_X</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GET_POS</sender>
   <signal>clicked()</signal>
   <receiver>pos_Y</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>24</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GET_CLICK</sender>
   <signal>clicked()</signal>
   <receiver>click_sum</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
